test/test-init.js

```diff
@@ -68,6 +71,21 @@ const seedMasterCopy = async (setup) => {
     return seed;
 };
 
+const Lbp = (setup) => new ethers.ContractFactory(LBP.abi, LBP.bytecode, setup.roles.root);
+
+const deployLBPFactory = async (setup) => {
+    const LBPFactory_Factory = new ethers.ContractFactory(LBPFactory.abi, LBPFactory.bytecode, setup.roles.root);
+    const lbpFactory = await LBPFactory_Factory.deploy(setup.vault.address);
+    return lbpFactory;
+}
+
+const deployVault = async (setup) => {
```

Comment:
let's move Vault and LBPFactory to a single class, we can call it balancer f.e.

Reply:
I also moved the LBP creating to the same file